### DISCUSSION TITLE
Small-scale deployment of Rook (e.g. 5 machines each of 4 core 8 GB memory)

### DISCUSSION THREAD

**Discussion by fzyzcjy** [rook/rook#10167] [General]
Hi thanks for the wonderful library! I wonder whether small-scale deployment of Rook is suitable? I have around 5 machines each of 4 core 8 GB memory, and need a storage provisioner. I hope it is not very memory hungry (totally <2GB), not of bad performance, and is easy to operate.

**Comment by travisn** [comment]
@fzyzcjy Three nodes is really the minimum requirement to run Rook. To keep storage stable and allow for the loss of at least one node, three mons and at least three OSDs are required (one per node). See [this doc](https://rook.github.io/docs/rook/v1.9/ceph-cluster-crd.html#cluster-wide-resources-configuration-settings) for the minimum recommended resources. But if the storage requirements are low, you could really even set the resource requirements lower.

**Reply by fzyzcjy** [reply]
Thank you!
